firmware/spade/src/rpi/CMakeLists.txt

```diff
@@ -12,6 +12,8 @@ target_compile_definitions(spade PRIVATE
   # PICO_DEFAULT_UART_RX_PIN=29
 )
 
+add_subdirectory("${CMAKE_CURRENT_LIST_DIR}/no-OS-FatFS/FatFs_SPI")
```

Comment:
if we just toss the fatfs stuff in our src this can go away...?

Reply:
I wasn't able to figure out how to remove this - the linker kept complaining about undefined references to functions defined in the library even if I explicitly included its CMakeLists.txt.